contracts/core/native_fallback/L2/NativeProver.sol

```diff
@@ -361,13 +350,45 @@ contract NativeProver is Ownable, INativeProver {
      * @param _rlpEncodedL1Header The encoded L1 header
      * @return L1 state root from the header
      */
-    function _validateL1BlockAndGetStateRoot(bytes calldata _rlpEncodedL1Header) internal view returns (bytes32) {
+    function _storeL1BlockAndGetStateRoot(bytes memory _rlpEncodedL1Header) internal returns (bytes32) {
```

Comment:
I believe we can get rid of _storeL1BlockAndGetStateRoot since the minimizing changes